etl/steps/data/grapher/wb/2024-12-17/bilateral_remittance.py

```diff
@@ -0,0 +1,68 @@
+"""Load a garden dataset and create a grapher dataset."""
+
+from etl.helpers import PathFinder, create_dataset
+
+# Get paths and naming conventions for current step.
+paths = PathFinder(__file__)
+
+
+indicator_title = "Estimated bilateral remittance flows"
+description_short = "The estimated bilateral remittance flows between countries. These estimates use migrant stocks, host country incomes, and origin country incomes."
+description_key = [
+    "Flows between India and Pakistan, Pakistan and India, Lebanon and Israel and vice versa, and Azerbaijan and Armenia and vice versa  are assumed to be zero given the political economy situations in these corridors."
+]
+description_from_producer = [
```

Comment:
This is also not formatting quite right at the moment, but I think using the garden version should be all good.
![Screenshot 2024-12-19 at 10 34 14](https://github.com/user-attachments/assets/7bddc3ef-eba9-4204-a218-e1d5b692f209)